app/src/main/res/values/strings.xml

```diff
@@ -77,6 +77,9 @@
     <string name="edit_profile_photo_message">Edit your profile picture</string>
     <string name="delete">Delete</string>
     <string name="new_photo">New Photo</string>
+    <string name="verified">Verified</string>
+    <string name="not_verified">User not verified</string>
```

Comment:
I guess something like "Your account hasn't been verified" would be better

Reply:
But too long. The layout will take two lines.

Reply:
Resend verification mail can be in the next line?

Reply:
Updated